scripts/post-comment.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# Copied from ember-data https://github.com/emberjs/data/blob/master/scripts/asset-size-tracking/src/post-comment.sh
```

Comment:
Can you make this either a perma link, or link to some specfici tag? Over time this link may become invalid...

Reply:
done